src/components/navbar/Navbar.jsx

```diff
@@ -4,7 +4,7 @@ const Navbar = () => {
     <nav className="navbar-container box-shadow">
     <div className="navbar">
       <div className="brand-name">
-        <a className="text-style-none font-lg" href="/pages/homepage/homepage.html"
+        <a className="text-style-none font-lg" 
```

Comment:
what is the use case of these anchor tags here?

Reply:
change later with Link tag to route pages.

Reply:
change later with Link tag to route pages.